src/domains/transaction/hooks/use-transaction-types.ts

```diff
@@ -38,6 +39,85 @@ type TransactionTypeProps = {
 	wallets?: Contracts.IReadWriteWallet[];
 };
 
+const core: Record<string, any> = {
+	transfer: {
+		type: CoreTransactionType.Transfer,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	vote: {
+		type: CoreTransactionType.Vote,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"second-signature": {
+		type: CoreTransactionType.SecondSignature,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"multi-signature": {
+		type: CoreTransactionType.MultiSignature,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"multi-payment": {
+		type: CoreTransactionType.MultiPayment,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	ipfs: {
+		type: CoreTransactionType.Ipfs,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"htlc-refund": {
+		type: CoreTransactionType.HtlcRefund,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"htlc-lock": {
+		type: CoreTransactionType.HtlcLock,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"htlc-claim": {
+		type: CoreTransactionType.HtlcClaim,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"delegate-registration": {
+		type: CoreTransactionType.DelegateRegistration,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+	"delegate-resignation": {
+		type: CoreTransactionType.DelegateResignation,
+		typeGroup: TransactionTypeGroup.Core,
+	},
+};
+
+const magistrate: Record<string, any> = {
```

Comment:
This whole object can be reduced to `magistrate` because the SDK now only looks for transactions with the type group and will also only identify them via `isMagistrate` and not any more specific.